src/components/Label/tests/Label.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Label from '../Label';
+
+describe('Label Suit', () => {
+
```

Comment:
Add a basic test case `describe('when default props are provided', () => {` and test it with required properties and expected output. For example, check Chip Component tests.